packages/openbadges-core/README.md

```
@@ -0,0 +1,119 @@
+# Run tests
+bun test
+
+# Watch mode
+bun test --watch
+```
+
+## Architecture
+
+This package follows a modular structure:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)